inc/classes/class_db_mysql.php

```diff
@@ -75,7 +75,7 @@ public function print_error($msg, $query_string_with_error)
      * @param array $match
      * @return int|mixed|string
      */
-    public function escape($match)
+    private function escape($match)
```

Comment:
I can't find any call to this function. Would vote for removal instead

Reply:
This is in use here: https://github.com/lansuite/lansuite/blob/master/inc/classes/class_db_mysql.php#L193

Reply:
well....shit. Did only a search for function calls, Not sure how we can search for callback uses :/